package.json

```diff
@@ -24,34 +24,42 @@
   "scripts": {
     "build": "tsc",
     "build-watch": "tsc -w",
-    "lint": "tslint --project ./tsconfig.json --config ./tslint.json",
+    "lint": "npx eslint 'src/**'",
     "prettier": "prettier --write 'src/**/*.ts'",
     "test": "jest",
     "generate-docs": "typedoc ./src/index.ts"
   },
   "engines": {
-    "node": ">=8.0"
+    "node": ">=18.0"
   },
   "dependencies": {
-    "node-fetch": "^2.6.0"
+    "node-fetch": "^2.6.7"
   },
   "devDependencies": {
-    "@types/deep-freeze": "^0.1.1",
-    "@types/jest": "^24.0.11",
-    "@types/nock": "^11.1.0",
-    "@types/node": "^12.6.9",
-    "@types/node-fetch": "^2.3.7",
-    "@zaiusinc/tslint-presets": "^1.0.1",
+    "@types/deep-freeze": "^0.1.2",
+    "@types/jest": "^28.1.8",
+    "@types/node": "^18.7.13",
+    "@types/node-fetch": "^2.6.2",
```

Comment:
What was the decision on upgrading to `v3.2.10`?

Reply:
there was no decision. to upgrade to v3.2.10 we would have to change packaging to EMS, which I'm not sure we can do. After reading about it, I think we should be fine to change the packaging, but I don't know enough about js world to be sure. 
if you're fine changing it, I'll give it a go

Reply:
one drawback with ESM is that jest has only experimental support for it https://jestjs.io/docs/ecmascript-modules

Reply:
@pawelziebaopti  - i thought the decision was to ignore the js-doc for the blank line between the comment and the args/return?  this should be consistently formatted across all our ts projects and it makes the diff unnecessarily noisy.

Reply:
i added a comment in the our eng team for the ESM switch.  i believe that will be a rather big change and not with the risk at this point in time just to support v3 1`node-fetch`.

Reply:
true, I upgraded eslint presets version and removed formatting changes